<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComSetupDlg</class>
 <widget class="QDialog" name="ComSetupDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QComboBox" name="listPort">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxCount">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Baud</string>
   </property>
  </widget>
  <widget class="QComboBox" name="listBaud">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxCount">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>221</width>
     <height>115</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QCheckBox" name="att1">
    <property name="text">
     <string>8-bits</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="att2">
    <property name="text">
     <string>Parity Check</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="att3">
    <property name="text">
     <string>1 stop bits</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="att4">
    <property name="text">
     <string>Hardware flow control</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="att5">
    <property name="text">
     <string>Software flow control</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>ComSetupDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
